hooks/tk-multi-publish2/basic/publish_renders.py

```diff
@@ -0,0 +1,510 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from collections import defaultdict
+import errno
+import os
+import re
+import textwrap
+
+import sgtk
+from sgtk.platform.qt import QtGui
+
+from Katana import NodegraphAPI
+
+HookBaseClass = sgtk.get_hook_baseclass()
+
+
+class RenderPublishWidgetItem(QtGui.QWidget):
+    """
+    A Widget to list all the versions of Look Files generated by a node.
+    """
+    def __init__(self, name, data, parent=None):
+        """
+        Initialize the class.
+
+        :param name: The node name.
+        :type name: str
+        :param data: The data used to populate this node.
+        :type data: dict
+        :type parent: QtGui.QWidget
+        """
+        super(RenderPublishWidgetItem, self).__init__(parent=None)
+        self.__name = name
+        layout = QtGui.QHBoxLayout()
+        self.setLayout(layout)
+        label = QtGui.QLabel(self.__name, parent=self)
+        layout.addWidget(label)
+        self._combo = QtGui.QComboBox(parent=self)
+        layout.addWidget(self._combo)
+        self._populate(self._combo, data)
+
+    @staticmethod
+    def _populate(combo, data):
+        """
+        Populate a combo box with the data provided.
+
+        :param combo:
+        :type combo: QtGui.QComboBox
+        :param data: The data to set on the combo box.
+        :type data: dict
+        """
+        work_paths = data.get("work_paths", [])
+        for index, path in enumerate(work_paths):
```

Comment:
Might not need to enumerate here as `index` is not used until line 63
```suggestion
        for path in work_paths:
```